tests/performance/load-tests.sh

```diff
@@ -5,6 +5,7 @@ set -e
 TIMESTAMP=$(date +%s)
 TEST_FOLDER=$(pwd)
 LOG_LEVEL="INFO"
+SETTING="oauth"
```

Comment:
IMHO, `SETTING` is too common name.
Name would be more readable if it expresses setting of what the value is representing.

Reply:
Fixed